ironfish/src/rpc/routes/wallet/createTransaction.ts

```diff
@@ -0,0 +1,181 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+ import { Asset } from '@ironfish/rust-nodejs'
+ import { BufferMap } from 'buffer-map'
+ import * as yup from 'yup'
+import { assertIsPriorityLevel, isPriorityLevel, PriorityLevel, PRIORITY_LEVELS } from '../../../memPool/feeEstimator'
+import { RawTransactionSerde } from '../../../primitives/rawTransaction'
+ import { CurrencyUtils } from '../../../utils'
+ import { NotEnoughFundsError } from '../../../wallet/errors'
+ import { ERROR_CODES, ValidationError } from '../../adapters/errors'
+ import { ApiNamespace, router } from '../router'
+ 
+ export type CreateTransactionRequest = {
+   fromAccountName: string
+   receives: {
+     publicAddress: string
+     amount: string
+     memo: string
+     assetId?: string
+   }[]
+   fee?: string
+   feePriorityLevel?: string
```

Comment:
this should take a fee rate not a fee priority level

Reply:
why you think it should be a fee rate? Other wallet like MetaMask, you specify the level. I can add fee rate as one of the request params but I dont want to limit the interface that user can not create transaction with fee level.